### ISSUE TITLE
MPGD intermittently fails check-overlap-geant4

### ISSUE THREAD

**Issue by veprbl** [eic/epic#109]
```
-------- WWWW ------- G4Exception-START -------- WWWW -------
*** G4Exception : GeomVol1002
      issued by : G4PVPlacement::CheckOverlaps()
Overlap with volume already placed !
          Overlap is detected for volume AV_586!InnerMPGDBarrel_Mod1_88#88!component1_1#1:90 (G4Box)
          apparently fully encapsulating volume AV_585!InnerMPGDBarrel_Mod1_87#87!component0_0#0:88 (G4Box) at the same level!
NOTE: Reached maximum fixed number -1- of overlaps reports for this volume !
*** This is just a warning message. ***
-------- WWWW -------- G4Exception-END --------- WWWW -------
```

_Originally posted by @veprbl in https://github.com/eic/epic/issues/107#issuecomment-1242530877_

**Comment by veprbl** [comment]
cc @mposik1983

**Comment by mposik1983** [comment]
Thanks for pointing this out. I am currently participating in a conference this week, if I have time I will try to look into this week, but may not be able to get to it until next week.

**Comment by veprbl** [comment]
Stopgap fix was applied in f45058389ca3dfdffe528bf4cfe2e473708db542
